source/loader/ze_ldrddi.cpp

```diff
@@ -55,6 +55,10 @@ namespace loader
 
         uint32_t total_driver_handle_count = 0;
 
+        if (!loader::context->coredriverSortingCompleted) {
```

Comment:
what if `zeDriverGet` is called again within `driverSorting` function?

Reply:
Fixed in latest, the recursion is now prevented.